### PR TITLE
Haskell: Configure Package Twice

### PR DESCRIPTION
**Pull request by sanssecours**
# Purpose

This update fixes a problem on the Travis build server, where `cabal build` would fail printing the following error message:

> Saved package config file is outdated
> …
> re-run the 'configure' command.

. This issue seems to have nothing to do with [caching](https://docs.travis-ci.com/user/caching), as I can reproduce the problem on my local machine too.

# Checklist

- [x] I checked all commit messages. The issue reference in commit 944c81a should be correct.
- [x] I ran all tests and everything went fine.
- [x] The PR also adds comments about the changes in commit b3533d1.

### CODE REVIEW

**Comment by sanssecours** [comment]
jenkins build all please

File: src/plugins/base64/README.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ Some file formats such as [YAML](http://yaml.org) already support Base64 encoded
 
 The diagram below shows how the Base64 conversion process works in conjunction with the YAML CPP plugin.
 
-![Tree](./base64/Base64.pdf)
+![Tree](./Base64.pdf)
```
Comment (markus2330) [inline src/plugins/base64/README.md]: Please also make it lower-case, see #1615

File: src/tools/kdb/test.cpp [hunk 2]
```diff
@@ -437,7 +437,7 @@ bool checkArgument (std::vector<std::string> const & arguments, std::string test
 {
 	return arguments.size () == 1 || find (arguments.begin (), arguments.end (), testname) != arguments.end ();
 }
-}
+} // namespace
```
Comment (markus2330) [inline src/tools/kdb/test.cpp]: Was this written manually by you or does the code formatter yield different results?

File: src/plugins/base64/README.md (hunk 1 above)
Comment (markus2330) [reply src/plugins/base64/README.md]: Actually, you can keep it that way. It seems like getting everything lower-case is impossible anyway. (e.g. README.md, CMakeLists.txt, ...)

File: src/tools/kdb/test.cpp (hunk 2 above)
Comment (sanssecours) [reply src/tools/kdb/test.cpp]: Clang-Format 6.0+ adds and corrects these comments if the option [`FixNamespaceComments`](https://github.com/ElektraInitiative/libelektra/blob/master/.clang-format#L36) is set to `true`.

Comment (markus2330) [reply src/tools/kdb/test.cpp]: see #1828

**Comment by ingwinlu** [comment]
jenkins build all please

**Comment by sanssecours** [comment]
jenkins build all please

**Comment by markus2330** [comment]
Why was this branch deleted without follow up? Some where nice things have been there.

**Comment by sanssecours** [comment]
> Why was this branch deleted without follow up?

I was (and still am) fed up with the current behavior/state of the Jenkins build server. I guess I will reopen this  PR after the situation improves.

**Comment by markus2330** [comment]
Sorry for these inconveniences.

I am sure the build server is now in much better hands thanks to @ingwinlu

**Comment by sanssecours** [comment]
I just pushed the remaining changes in this PR to the `master` branch.

**Pull request closed without merging**